include/openssl/evp.h

```diff
@@ -1739,6 +1739,7 @@ int EVP_PKEY_CTX_set_mac_key(EVP_PKEY_CTX *ctx, const unsigned char *key,
  * Method handles all operations: don't assume any digest related defaults.
  */
 # define EVP_PKEY_FLAG_SIGCTX_CUSTOM     4
+
```

Comment:
nit: necessary change?

Reply:
Nope... we can clean it up on the next rebase (or a subsequent clean-up commit). It was probably left over from when I reverted any FIPS-checksum-impacting changes.